vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -0,0 +1,220 @@
+                                sub_tests: ['microbenchmark'],
+                                scheduler: 'master-vnodes'
+                            ],
+                            [
+                                job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_arm64-write',
+                                region: 'us-east-1',
+                                versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'],
+                                sub_tests: ['microbenchmark'],
+                                scheduler: 'master-vnodes'
+                            ],
+                            [
+                                job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_x86_64',
+                                region: 'us-east-1',
+                                versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'],
+                                sub_tests: ['microbenchmark'],
+                                scheduler: 'master-vnodes'
+                            ],
+                            [
+                                job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_x86_64-write',
+                                region: 'us-east-1',
+                                versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'],
+                                sub_tests: ['microbenchmark'],
+                                scheduler: 'master-vnodes'
+                            ],
+                            [
+                                job_name: 'scylla-master/perf-regression/perf-regression-predefined-throughput-steps-sanity-vnodes',
+                                region: '',
+                                versions: [],
+                                sub_tests: [],
+                                scheduler: 'master-daily'
+                            ],
+                            [
+                                job_name: 'scylla-staging/yulia/performance/perf-regression-predefined-throughput-steps-sanity-vnodes',
+                                region: 'us-east-1',
+                                versions: ['master'],
+                                sub_tests: [''"test_mixed_gradual_increase_load"''],
+                                scheduler: 'master-daily'
+                            ],
+                            // Tablets
+                            [
+                                job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-tablets',
+                                region: 'us-east-1',
+                                versions: ['2025.1', '2025.2', 'master'],
+                                sub_tests: ['test_read_gradual_increase_load', 'test_mixed_gradual_increase_load'],
+                                scheduler: 'master-tablets'
```

Comment:
if we still want to run this on a weekly basis, it should move to `master-vnodes`

or we need to define a new group, and put one more schedule for it

Reply:
changed schedulers to:
- master-weekly
- master-3weeks
- master-daily